packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -109,26 +109,27 @@ const ActivityDetails = (props: Props) => {
     )
   }, [templateId, submitting, submitMutation, onError, onCompleted])
 
-  if (!selectedTemplate) {
```

Comment:
`makeTemplateDescription` was being called conditionally because of this, so we need to reorder some of this logic.